src/test/java/org/swisspush/redisques/RedisQuesTest.java

```diff
@@ -323,6 +327,7 @@ public void getQueuesCountFiltered(TestContext context) {
                 context.assertEquals(OK, message.result().body().getString(STATUS));
                 asyncEnqueue.countDown();
             });
+            Thread.sleep(10);
```

Comment:
Why is this necessary? The test will wait until the countDown has gone from 50 to 0